data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -1 +1,552 @@
+        name= _ "Towards Thoria"
+        name_completed= _ "Towards Thoria"
+        description_completed= _ "Found the secret in <i>A Choice must be Made</i>"
+        icon="units/unknown-unit.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/merfolk/warrior.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S19A_Fires_of_Pride"
+        name= _ "Fires of Pride"
+        name_completed= _ "Fires of Pride"
+        description_completed= _ "Found the legendary Sword of Fire!"
+        icon="items/flame-sword-bare.png~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="items/flame-sword.png~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S19B_Forgotten_Armament"
+        name= _ "Forgotten Armament"
+        name_completed= _ "Forgotten Armament"
+        description_completed= _ "Found the legendary Void Armor!"
+        icon="units/unknown-unit.png~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="icons/breastplate2.png~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_19C_Armageddon"
+        name= _ "Armageddon"
+        name_completed= _ "Armageddon"
+        description= _ "Defeat Keh Ohn"
+        description_completed= _ "Defeated Keh Ohn, the Lord of the Drakes"
+        icon="units/unknown-unit.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="attacks/fire-breath-drake.png~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_19C_At_Deaths_Door"
+        name= _ "At Death's Door"
+        name_completed= _ "At Death's Door"
+        description= _ "Find Warven in <i>Cliffs of Thoria</i>"
+        description_completed= _ "Found Warven the Sergeant in <i>Cliffs of Thoria</i>!"
+        icon="units/unknown-unit.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/human-loyalists/lieutenant-die-9.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_20B_Lore_of_Ancients"
+        name= _ "Lore of Ancients"
+        description= _ "Found the Book containing the knowledge of Woses!"
+        icon="icons/book.png~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S22_Aid_from_the_Skies"
+        name= _ "Aid from the Skies"
+        description= _ "Gryphons have decided to join your plight!"
```

Comment:
Not sure if "plight" is used correctly here... "plight" doesn't mean "fight", it's more like "suffering"... maybe "Gryphons have decided to aid you in your plight!" instead?
(Also this sounds more like a `description_completed` than a `description`)